zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -193,6 +200,22 @@ public ZeppelinServer() throws Exception {
     notebook.addNotebookEventListener(heliumApplicationFactory);
     notebook.addNotebookEventListener(notebookWsServer.getNotebookInformationListener());
     this.interpreterService = new InterpreterService(conf, interpreterSettingManager);
+
+    // Register MBean
+    MBeanServer mBeanServer = ManagementFactory.getPlatformMBeanServer();
```

Comment:
can or should we do this only the JMX is enabled?

Reply:
Fixed